src/session.ts

```diff
@@ -62,7 +62,6 @@ export class DebugSession implements IDebugger.ISession {
     }
 
     this._connection.iopubMessage.connect(this._handleEvent, this);
-    this._connectionChanged.emit(this._connection);
```

Comment:
Is this signal emitted somewhere else? Or does that mean it could be removed from the interfaces as well?

Reply:
It doesn't look like it's used anywhere, so this PR removes it from the interface in `token.ts` as well.

Reply:
Yes, it was just rebased to remove it.

Reply:
Gotcha, I couldn't figure out why I couldn't find it 😂